### ISSUE TITLE
Related Objects - Child Folder relations not displaying folder icon

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/hpi#1764]
On the stage for edge1 and release1 (Documentum), when viewing a related objects section configured for `children` relationships, the folder icon is no longer appearing for subfolders.

**edge1**
![image](https://user-images.githubusercontent.com/1377228/45970627-9fe44080-bffc-11e8-920f-017045ec0898.png)

**edge2**
![image](https://user-images.githubusercontent.com/1377228/45970641-ab376c00-bffc-11e8-8ad0-97fce4ae9120.png)

Note that on edge1, the folders do display properly in search results as well as in non-children related objects (ex: folder tags).

**Comment by grahamsinger** [comment]
@gsteimer looks like it could be a backend issue as all mimetypes seem to be coming back as `application/octet-stream`

**Comment by gsteimer** [comment]
Hmm... but if I expand the folder, the Word icons are appearing fine:

![image](https://user-images.githubusercontent.com/1377228/45974358-bd1e0c80-c006-11e8-8fbc-d17064dfca1b.png)

Here's another example of a folder with just one document in it:

![image](https://user-images.githubusercontent.com/1377228/45974401-d6bf5400-c006-11e8-8727-baf0208114c2.png)

**Comment by grahamsinger** [comment]
@gsteimer Sorry, I meant specifically for (all) of the folders.

**Comment by gsteimer** [comment]
Yeah - it must be something to do with getting child folders since it works for documents

![image](https://user-images.githubusercontent.com/1377228/45974868-4550e180-c008-11e8-8918-97f2b5e0c9a6.png)

**Comment by gsteimer** [comment]
I checked a related objects section that runs a search - specifically to find related claim folders on a policy.  This one works fine and displays a folder icon.  The `mimeType` is null:

![image](https://user-images.githubusercontent.com/1377228/45975091-e6d83300-c008-11e8-8163-c94ee0e3c180.png)

Perhaps something changed on the back end and now instead of null, `getChildren` is setting the octet stream mime type...
